Lib/fontTools/varLib/cff.py

```diff
@@ -205,11 +205,10 @@ def merge_PrivateDicts(top_dicts, vsindex_dict, var_model, fd_map):
 				try:
 					values = [pd.rawDict[key] for pd in pds]
 				except KeyError:
-					del private_dict.rawDict[key]
```

Comment:
are you sure you want to delete this line?

Reply:
Yes. The VF will have the item with the values of the default master. No blends. I think that's better than dropping it entirely.